src/stactools/palsar2_scansar/constants.py

```diff
@@ -0,0 +1,153 @@
+
+SCANSAR_REVISION = "E"
+
+SCANSAR_LINKS = [
+    Link(
+        rel="documentation",
+        target=("https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm"),
+        media_type="application/pdf",
+        title="PALSAR-2 ScanSAR NRB product description",
+        extra_fields={"description": "Also includes data usage information"},
+    ),
+    Link(
+        rel="license",
+        target="https://earth.jaxa.jp/policy/en.html",
+        title="JAXA Terms of Use of Research Data",
+    ),
+]
+
+
+SCANSAR_PALSAR_PLATFORMS = ["ALOS2"]
+SCANSAR_PALSAR_INSTRUMENTS = ["PALSAR-2"]
+SCANSAR_PALSAR_GSD = 25  # meters
+SCANSAR_PALSAR_EPSG = 4326
+SCANSAR_PALSAR_CF = "83.0 dB"
+
+SCANSAR_SAR: Dict[str, Any] = {
+    "looks_range": [1],
+    "product_type": ["GRD"],
+    "looks_azimuth": [1],
+    "polarizations": [sar.Polarization.HH, sar.Polarization.HV],
+    "frequency_band": [sar.FrequencyBand.L],
+    "instrument_mode": ["WBD", "WBS"],
+    "center_frequency": [1.2575],
+    "resolution_range": [19],
+    "resolution_azimuth": [25.9],
+    "pixel_spacing_range": [8.583],
+    "observation_direction": [sar.ObservationDirection.RIGHT],
+    "pixel_spacing_azimuth": [325.003],
+    "looks_equivalent_number": [2.7],
+}
+
+SCANSAR_POLARIZATIONS = {
+    "HH_SLP": Band.create(
+        name="HH",
+        description="HH band: horizontal transmit and horizontal receive",
+    ),
+    "HV_SLP": Band.create(
+        name="HV",
+        description="HV band: horizontal transmit and vertical receive",
+    ),
+}
+
+SCANSAR_ASSETS: Dict[str, AssetDefinition] = {
+    "HH_SLP": AssetDefinition(
+        {
+            "title": "HH",
+            "type": pystac.MediaType.COG,
+            "description": "HH polarization backscattering coefficient, 16-bit DN.",
+            "role": "data",
+        }
+    ),
+    "HV_SLP": AssetDefinition(
+        {
+            "title": "HV",
+            "type": pystac.MediaType.COG,
+            "description": "HV polarization backscattering coefficient, 16-bit DN.",
+            "role": "data",
+        }
+    ),
+    "LIN": AssetDefinition(
+        {
+            "title": "linci",
+            "type": pystac.MediaType.COG,
+            "description": "Local incidence angle (degrees).",
+            "role": ["data", "local-incidence-angle"],
+        }
+    ),
+    "MSK": AssetDefinition(
+        {
+            "title": "mask",
+            "type": pystac.MediaType.COG,
+            "description": "Quality Mask",
+            "role": ["data", "data-mask"],
+        }
+    ),
+    "summary": AssetDefinition(
+        {
+            "title": "summary",
+            "type": pystac.MediaType.XML,
+            "description": "CARD4L Metadata",
+            "role": "metadata",
+        }
+    ),
+    "kml": AssetDefinition(
+        {
+            "title": "kml",
+            "type": pystac.MediaType.XML,
```

Comment:
Double-checking, is this the preferred content-type for KML?

Reply:
Good catch, updated to IANA type for KML `application/vnd.google-earth.kml+xml`